data/json/construction.json

```diff
@@ -7867,5 +7867,62 @@
     "components": [ [ [ "rock_large", 22 ] ] ],
     "pre_terrain": "t_dirt",
     "post_terrain": "t_rockyobstacle"
+  },
+  {
+    "type": "construction",
+    "id": "constr_brick_oven_prep",
+    "group": "build_brick_oven",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "time": "2 h",
+    "tools": [ [ [ "shovel", -1 ], [ "primitive_shovel", -1 ] ] ],
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ] ],
+    "components": [ [ [ "fire_brick", 30 ] ], [ [ "clay_lump", 5 ] ], [ [ "water", 2 ], [ "water_clean", 2 ] ] ],
+    "pre_flags": "DIGGABLE",
+    "pre_special": "check_empty",
+    "post_terrain": "t_brick_oven_base",
+    "activity_level": "ACTIVE_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_brick_oven_struct",
+    "group": "build_brick_oven",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 5 ] ],
+    "time": "5 h",
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ] ],
+    "components": [ [ [ "fire_brick", 60 ] ], [ [ "clay_lump", 30 ] ], [ [ "water", 15 ], [ "water_clean", 15 ] ] ],
+    "pre_terrain": "t_brick_oven_base",
+    "post_terrain": "t_brick_oven_struct",
+    "activity_level": "BRISK_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_brick_oven_finisher",
+    "group": "build_brick_oven",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 3 ] ],
+    "time": "3 h",
+    "pre_note": "Carve a thick wooden door for the brick oven.",
+    "qualities": [ [ { "id": "AXE", "level": 2 } ], [ { "id": "CHISEL_WOOD", "level": 1 } ] ],
+    "components": [ [ [ "log", 1 ] ] ],
```

Comment:
Couldn't you make it out of planks? Maybe it'd be more convenient to have the door be an actual item with a recipe and then have the consrtuction just require the door item.

Reply:
>Couldn't you make it out of planks?

That's a distinct 'maybe'. The book I used as reference does not elaborate on the construction of wooden doors. It does have a singular image of one (ch.1 pg 18), which appears to be several joined planks with a crossbeam and crude handle.

I think it could be added, I have nothing against it, I just don't feel the need/effort for a third option. I'll leave this conversation unresolved so you/the other mergers can decide if that should be added.

>Maybe it'd be more convenient to have the door be an actual item with a recipe and then have the consrtuction just require the door item.

Personally I do not like these very "bespoke" items with single purposes. 

Implementing it that way would mean 2 more entries in the crafting menu, and it'd still need an entry in the construction menu to change the terrain! More json, more definitions, little to gain IMO.